varcode/variant_collection.py

```diff
@@ -161,10 +124,10 @@ def effects_to_string(self, *args, **kwargs):
         Arguments are passed on to variant_effects(*args, **kwargs).
         """
         lines = []
-        for variant_effect in self.variant_effects(*args, **kwargs):
+        for annotation in self.annotations(*args, **kwargs):
```

Comment:
Works for me!
On Feb 20, 2015 2:07 PM, "Alex Rubinsteyn" <notifications@github.com> wrote:

> In varcode/variant_collection.py
> <https://github.com/hammerlab/varcode/pull/20#discussion_r25094042>:
>
> > @@ -161,10 +124,10 @@ def effects_to_string(self, *args, **kwargs):
> >          Arguments are passed on to variant_effects(*args, **kwargs).
> >          """
> >          lines = []
> > -        for variant_effect in self.variant_effects(*args, **kwargs):
> > +        for annotation in self.annotations(*args, **kwargs):
>
> With the new names, what do you think of effect_summary for the to_string
> method?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/varcode/pull/20/files#r25094042>.
>